packages/components/psammead-story-promo/src/index.jsx

```diff
@@ -114,6 +116,29 @@ export const Summary = styled.p`
   }
 `;
 
+export const Link = styled.a`
+  position: static;
+  color: ${C_SHADOW};
+  text-decoration: none;
+
+  &:before {
+    bottom: 0;
+    content: '';
+    left: 0;
+    overflow: hidden;
+    position: absolute;
+    right: 0;
+    top: 0;
+    white-space: nowrap;
+    z-index: 1;
+  }
+
+  &:hover,
+  &:focus {
+    text-decoration: underline;
```

Comment:
Are we having visited states?

Reply:
Yes, but this issue is just about `hover` and `focus`.

Reply:
Ok fine, for the swarm on this component it should be code complete, otherwise we will need to re-swarm...

Reply:
I added the visited state to the link.